src/utils.c

```diff
@@ -1,26 +1,21 @@
 #include <postgres.h>
-#include <fmgr.h>
 #include <catalog/pg_type.h>
-#include <catalog/pg_trigger.h>
 #include <catalog/namespace.h>
 #include <catalog/pg_inherits.h>
 #include <catalog/indexing.h>
 #include <access/htup.h>
 #include <access/htup_details.h>
 #include <access/heapam.h>
 #include <access/genam.h>
-#include <nodes/nodes.h>
 #include <nodes/makefuncs.h>
 #include <parser/scansup.h>
-#include <utils/guc.h>
-#include <utils/date.h>
-#include <utils/datetime.h>
 #include <utils/lsyscache.h>
 #include <utils/syscache.h>
 #include <utils/relcache.h>
 #include <utils/fmgroids.h>
 #include <catalog/pg_cast.h>
 #include <parser/parse_coerce.h>
+#include <fmgr.h>
```

Comment:
Why did this get moved down?

Reply:
Was trying to clean up unused includes and this got moved in the process.